launch/launch/launch_context.py

```diff
@@ -170,6 +185,11 @@ def launch_configurations(self) -> Dict[Text, Text]:
         """Getter for launch_configurations dictionary."""
         return self.__launch_configurations
 
+    @property
+    def environment(self) -> Dict[Text, Text]:
+        """Getter for environment variables dictionary."""
+        return self.__environment
```

Comment:
I suppose we could push/pop `os.environ` directly.. then it should be backwards compatible :thinking:

Reply:
see [3d2a695](https://github.com/ros2/launch/pull/601/commits/3d2a6950cb0ae0424e273838968e2ff495ba68a3)